apps/nextjs/src/components/AppComponents/Chat/drop-down-section/chat-section.tsx

```diff
@@ -28,11 +29,20 @@ const ChatSection = ({
         $position="relative"
         $display={["none", "flex"]}
       >
-        <ModifyButton
-          sectionTitle={sectionTitle(objectKey)}
-          sectionPath={objectKey}
-          sectionValue={value}
-        />
+        {objectKey === "additionalMaterials" && value === "None" ? (
```

Comment:
Just sense checking: Is `None` the default?

Reply:
Yeah a lesson plan comes back with 'none' or nothing.
From the ticket : 

> When there is no additional materials section at all, no change to the current working.
> Additional materials = None - Button would say ‘Add additional materials’ instead of ‘modify’

Reply:
Ah great. Thanks